default-templates/server/index.html

```diff
@@ -45,7 +45,7 @@ <h2>Server info</h2>
       <dd>{{serverDescription}}</dd>
       {{/if}}
       <dt>Details</dt>
-      <dd>Running on <a href="https://solid.mit.edu/">Solid {{serverVersion}}</a> (<a href="https://solid.inrupt.com/docs">Documentation</a>)</dd>
+      <dd>Running on <a href="https://www.w3.org/community/solid/wiki/">Solid {{serverVersion}}</a></dd>
```

Comment:
Looks good, maybe add https://www.w3.org/community/solid/wiki/Main_Page as the 'Documentation' link? I think there's rough consensus about that on the parent issue.

Reply:
Further commits pending https://github.com/solid/node-solid-server/issues/1152#issuecomment-476639962